src/librustc/session/config.rs

```diff
@@ -974,6 +978,7 @@ pub fn build_session_options(matches: &getopts::Matches) -> Options {
     let parse_only = debugging_opts.parse_only;
     let no_trans = debugging_opts.no_trans;
     let treat_err_as_bug = debugging_opts.treat_err_as_bug;
+    let mir_opt_level = debugging_opts.mir_opt_level.unwrap_or(1);
```

Comment:
[I was considering](https://github.com/rust-lang/rust/pull/31448) a default of `1 + -C opt-level`. This way the level of MIR “optimisations” would automatically get attached to the `-O` and `-C opt-level` flags as well.